tests/integration/test_agent.py

```diff
@@ -282,3 +281,15 @@ def test_browse_internet(http_server, current_test_name: str):
         assert 'OpenDevin is all you need!' in last_observation.content
     elif isinstance(last_observation, AgentDelegateObservation):
         assert 'OpenDevin is all you need!' in last_observation.outputs['content']
+
+
+def test_dummy_agent(current_test_name: str):
```

Comment:
https://github.com/OpenDevin/OpenDevin/pull/1316#discussion_r1581732909 @li-boxuan Why do you think it's a bad idea to add this as a pytest?

Reply:
I don't remember. Things might have changed and maybe it's a good time now.

Reply:
You need to do ```@pytest.mark.skipif(os.environ.get('DEFAULT_AGENT') != 'DummyAgent')``` for this test

and also skip DummyAgent for every other test.